src/System.Reflection.Metadata/tests/Configurations.props

```diff
@@ -4,7 +4,7 @@
     <BuildConfigurations>
       netcoreapp;
       netfx;
-      <!-- netstandard; -->
+      <!-- uap; -->
```

Comment:
Is there something tracking this? Should we instead delete it?

Reply:
I think we can safely delete this.